dash/_callback_context.py

```diff
@@ -96,5 +125,23 @@ def record_timing(name, duration=None, description=None):
 
         setattr(flask.g, "timing_information", timing_information)
 
+    @property
+    @has_context
+    def using_args_grouping(self):
+        """
+        Return True if this callback is using dictionary or nested groupings for
+        Input/State dependencies.
```

Comment:
or interleaved inputs and state - does that trigger this flag?

Reply:
Good point. It does now (9f41b86)